app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Sign in, please:</h1>
+
```

Comment:
It would be nice to have some error handling in this view. When I entered an incorrect email/password combination, I wasn't sure if it was working or not. An "Incorrect email/password combo" error message would improve the user experience.